pkg/client/powervs_client.go

```diff
@@ -261,8 +267,20 @@ func NewMinimalPowerVSClient(ctrlRuntimeClient client.Client) (Client, error) {
 		return nil, err
 	}
 
+	vpcOptions := &vpcv1.VpcV1Options{
+		Authenticator: authenticator,
+	}
+	if endpoints[customVpcEndpointName] != "" {
+		vpcOptions.URL = fmt.Sprintf("%s/v1", endpoints[customVpcEndpointName])
+	}
+	vpcClient, err := vpcv1.NewVpcV1(vpcOptions)
+	if err != nil {
+		return nil, err
+	}
+
 	return &powerVSClient{
```

Comment:
I think we need not update the minimal client with vpcClient because minimalClient is only used to fetch the Power VS Service instance details https://github.com/Karthik-K-N/machine-api-provider-powervs/blob/da14713a9e2c5b27bb00383a4f5d08b15908bc58/pkg/actuators/machine/machine_scope.go#L80-L87, It does not deal anything with VPC.

Reply:
Okay, sounds good. I added this bit by following my previous mods for RC override. Will drop...